envVariables.js

```diff
@@ -123,6 +123,18 @@ let enviromentVariables = {
 		message: 'Required api doc url',
 		optional: false,
 	},
+	INTERFACE_SERVICE_URL: {
+		message: 'Interface service url required',
+		optional: false,
+	},
+	USER_SERVICE_NAME: {
```

Comment:
@Prajwal17Tunerlabs please update the variable names here as well

Reply:
env variable names are updated in envVariable.js file.